src/Fixer/Operator/OperatorLinebreakFixer.php

```diff
@@ -0,0 +1,262 @@
+                ->setDefault($this->position)
+                ->setAllowedValues(['beginning', 'end'])
+                ->getOption(),
+        ]);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        parent::configure($configuration);
+
+        if (isset($configuration['only_booleans'])) {
+            $this->onlyBooleans = $configuration['only_booleans'];
+        }
+        if (isset($configuration['position'])) {
+            $this->position = $configuration['position'];
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return true;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens)
+    {
+        if ('beginning' === $this->position) {
+            $this->fixMoveToTheBeginning($tokens);
+        } else {
+            $this->fixMoveToTheEnd($tokens);
+        }
+    }
+
+    private function fixMoveToTheBeginning(Tokens $tokens)
+    {
+        for ($index = 0; $index < $tokens->count(); ++$index) {
+            $indices = $this->getOperatorIndices($tokens, $index);
+            if (null === $indices) {
+                continue;
+            }
+
+            $nextIndex = $tokens->getNextMeaningfulToken(max($indices));
+            for ($i = $nextIndex - 1; $i > $index; --$i) {
+                if ($tokens[$i]->isWhitespace() && 1 === Preg::match('/\R/u', $tokens[$i]->getContent())) {
+                    $operators = $this->getReplacementsAndClear($tokens, $indices, -1);
+                    $tokens->insertAt($nextIndex, array_merge($operators, [new Token([T_WHITESPACE, ' '])]));
+
+                    break;
+                }
+            }
+            $index = $nextIndex;
+        }
+    }
+
+    private function fixMoveToTheEnd(Tokens $tokens)
+    {
+        for ($index = $tokens->count() - 1; $index > 0; --$index) {
+            $indices = $this->getOperatorIndices($tokens, $index);
+            if (null === $indices) {
+                continue;
+            }
+
+            $prevIndex = $tokens->getPrevMeaningfulToken(min($indices));
+            for ($i = $prevIndex + 1; $i < $index; ++$i) {
+                if ($tokens[$i]->isWhitespace() && 1 === Preg::match('/\R/u', $tokens[$i]->getContent())) {
+                    $operators = $this->getReplacementsAndClear($tokens, $indices, 1);
+                    $tokens->insertAt($prevIndex + 1, array_merge([new Token([T_WHITESPACE, ' '])], $operators));
+
+                    break;
+                }
+            }
+            $index = $prevIndex;
+        }
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     *
+     * @return null|int[]
+     */
+    private function getOperatorIndices(Tokens $tokens, $index)
+    {
+        if (null === $this->operators) {
```

Comment:
can `$this->operators` be resolved in the `configure` method?

Reply:
Done